docs/osquery/exported-fields-reference.asciidoc

```diff
@@ -80,6 +80,10 @@ For more information about osquery tables, see the https://osquery.io/schema[osq
 
 * _systemd_units.active_state_ - The high-level unit activation state, i.e. generalization of SUB
 
+*activity* - keyword, number.long
+
+* _unified_log.activity_ - the activity ID associate with the entry
```

Comment:
Some descriptions have periods at the end and some don't. In a future iteration, it might be good to set a standard for this so that all descriptions are punctuated the same. :)